start_avm.sh

```diff
@@ -56,7 +56,7 @@ else
     NETWORK="-netdev user,id=hostnet0"
 fi
 if [ $VHOST_USER == 1 ]; then
-    /usr/libexec/vhost-user-vsock --socket ${CVD_BASE_DIR}/qemu/vhost-user-vsock.sock --uds-path ${CVD_BASE_DIR}/qemu/vhost-user-vsock.uds &
```

Comment:
Why do we expect `vhost-user-vsock` to be installed in `/usr/bin`? Other vhost-user services are installed in `/usr/libexec` on Fedora/RHEL.

Reply:
well, the one from your copr installs to /usr/bin
[root@3f53f8c09907 /]# dnf info vhost-user-vsock
Installed Packages
Name         : vhost-user-vsock
Version      : 0.1.0
Release      : 1.fc38
Architecture : x86_64
Size         : 2.4 M
Source       : rust-vhost-user-vsock-0.1.0-1.fc38.src.rpm
Repository   : @System
From repo    : copr:copr.fedorainfracloud.org:slp:qemu-android
[root@3f53f8c09907 /]# rpm -ql vhost-user-vsock
/usr/bin/vhost-user-vsock

Reply:
Fair enough. We'll change this again when vhost-user-vsock is properly packaged.